Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -132,6 +143,153 @@ TEST(BloomFilterTest,
   }
 }
 
+class BloomFilterGoldenTest : public ::testing::Test {
+ public:
+  void RunGoldenTest(const std::string& test_file) {
```

Comment:
It looks like `RunGoldenTest()` can be declared `static` (after moving `JsonReader reader` to a local variable, as recommended below). Please make this change, if possible.